syncer/metrics/metrics.go

```diff
@@ -219,76 +213,51 @@ var (
 
 // RegisterMetrics registers metrics.
 func RegisterMetrics(registry *prometheus.Registry) {
-	registry.MustRegister(binlogReadDurationHistogram)
-	registry.MustRegister(binlogEventSizeHistogram)
-	registry.MustRegister(binlogEvent)
-	registry.MustRegister(conflictDetectDurationHistogram)
-	registry.MustRegister(addJobDurationHistogram)
-	registry.MustRegister(dispatchBinlogDurationHistogram)
-	registry.MustRegister(skipBinlogDurationHistogram)
-	registry.MustRegister(addedJobsTotal)
-	registry.MustRegister(finishedJobsTotal)
-	registry.MustRegister(queueSizeGauge)
-	registry.MustRegister(sqlRetriesTotal)
-	registry.MustRegister(binlogPosGauge)
-	registry.MustRegister(binlogFileGauge)
-	registry.MustRegister(txnHistogram)
-	registry.MustRegister(stmtHistogram)
-	registry.MustRegister(queryHistogram)
-	registry.MustRegister(syncerExitWithErrorCounter)
-	registry.MustRegister(replicationLagGauge)
-	registry.MustRegister(remainingTimeGauge)
-	registry.MustRegister(unsyncedTableGauge)
-	registry.MustRegister(shardLockResolving)
-	registry.MustRegister(heartbeatUpdateErr)
-}
-
-// InitStatusAndMetrics register prometheus metrics and listen for status port.
-func InitStatusAndMetrics(addr string) {
```

Comment:
seems a unused function

Reply:
maybe this used in `dm-syncer` before 😂
I don't know when it was deleted